gocd/gocd.go

```
@@ -206,6 +217,11 @@ func pipelineGET(gocdURL string, pipelineName string) (pipeline Pipeline, etag s
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode != 200 {
+		err = fmt.Errorf("Bad response code: %d", resp.StatusCode)
+		return pipeline, etag, err
+  }
+	
```

Agreed, this was just to prevent the test from failing silently.